### DISCUSSION TITLE
Question: Is there any way to get the contact of whom the user shared with?

### DISCUSSION THREAD

**Discussion by AngelusDarkness** [videokit-ai/natshare#114] [Q&A] [answered]
Hi first of all, thanks for doing this plugin, is super useful for us.

We have an issue when trying to share using iMessage on iOS where video + text are not share together (only the video asset) like on other social media.

One solution could be to send two consecutive messages one with the video and other with the text, however I'm not finding a way to get information about the contact chosen by the user.

Do you know if this is something that can be achieve through this plugin?

Thanks,
Pablo

**Comment by olokobayusuf** [comment, marked as answer]
Hey there @AngelusDarkness , you seem to be asking two questions between the title and the body of this discussion. First on the title:

> Is there any way to get the contact of whom the user shared with?

No, this isn't possible. I don't believe either of the platforms support this, as there would be privacy concerns with sharing such information with apps.

> We have an issue when trying to share using iMessage on iOS where video + text are not share together (only the video asset) like on other social media.

This depends solely on the receiving app. The receiving app chooses what parts of the payload to keep and what parts to discard.

**Reply by AngelusDarkness** [reply]
Thank you very much for your reply!
